### ISSUE TITLE
Option to apply WarningList to non-IDS attributes

### ISSUE THREAD

**Issue by nephdev** [MISP/MISP#3970]
I think that there should be some setting in "Server Settings & Maintenance" which would allow us to apply warning list also to non IDS attributes. 

While analyzing malicious behavior analyst can see many attributes. Some of them are easily described as false positive (google.com, etc.). However some others are harder to detect and it would be great if we could see that specific attribute is possible false positive even without setting "for IDS" flag to true.

**Comment by iglocska** [comment]
Makes sense. Right now the warninglists are somewhat limited in use to the one use-case we've had in mind, the idea would be to enhance this by having warninglists that serve different purposes (one would be purely warnings, some could indeed be applied to attributes irregardless of to_ids settings, etc)
